### PR TITLE
meta: factory updates

### PR DESCRIPTION
**Pull request by DarkGhost7**

### CODE REVIEW

File: data/meta/tokens/1/0x051d7e5609917Bd9b73f04BAc0DED8Dd46a74301.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sBTCv2 Pool",
+  "symbol": "crvWSBTC",
+  "description": "This token represents a Curve liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This liquidity pool contains sBTC and wBTC.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0x051d7e5609917Bd9b73f04BAc0DED8Dd46a74301.json:5]: `This liquidity pool contains sBTC and wBTC and serves as a base pool for other Bitcoin pools.`

Comment (DarkGhost7) [reply data/meta/tokens/1/0x051d7e5609917Bd9b73f04BAc0DED8Dd46a74301.json]: fixed

File: data/meta/tokens/1/0x5ca0313D44551e32e0d7a298EC024321c4BC59B4.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve bLUSD Pool",
+  "symbol": "bLUSDLUSD3-f",
+  "description": "This token represents a v2 Curve liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This crypto pool contains bLUSD, LUSD3CRV-f the liquity metapool composed of: LUSD, DAI, USDC, USDT. Please be aware that as crypto pools are composed of differently-priced assets, they are subject to impermanent loss.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0x5ca0313D44551e32e0d7a298EC024321c4BC59B4.json:5]: Edits to the third sentence, along with adding one about bLUSD:

This crypto pool contains bLUSD along with LUSD3CRV-f, the base LUSD pool composed of LUSD, DAI, USDC, and USDT. For more info regarding bLUSD, check [here](https://www.chickenbonds.org/lusd-bonds).

File: data/meta/tokens/1/0x48fF31bBbD8Ab553Ebe7cBD84e1eA3dBa8f54957.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdAG-AG Pool",
+  "symbol": "sdAGAG-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains ANGLE and sdANGLE.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0x48fF31bBbD8Ab553Ebe7cBD84e1eA3dBa8f54957.json:5]: This pool contains ANGLE and sdANGLE, a tokenized veANGLE from StakeDAO.

File: data/meta/tokens/1/0x79e281bc69a03dabccd66858c65ef6724e50aebe.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdYFI-YFI Pool",
+  "symbol": "sdYFI",
+  "description": "This token represents a Curve liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains YFI and sdYFI, a Staked YFI token from StakeDAO.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0x79e281bc69a03dabccd66858c65ef6724e50aebe.json]: This pool contains YFI and sdYFI, a tokenized veYFI from StakeDAO.

File: data/meta/tokens/1/0x8c524635d52bd7b1Bd55E062303177a7d916C046.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdFXS-FXS Pool",
+  "symbol": "sdFXSFXS-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains sdFXS and FXS.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0x8c524635d52bd7b1Bd55E062303177a7d916C046.json:5]: This pool contains FXS and sdFXS, a tokenized veFXS from StakeDAO.

File: data/meta/tokens/1/0xf7b55C3732aD8b2c2dA7c24f30A69f55c54FB717.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdCRV-CRV Pool",
+  "symbol": "sdCRVCRV-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains sdCRV and CRV.",
```
Comment (dudesahn) [inline data/meta/tokens/1/0xf7b55C3732aD8b2c2dA7c24f30A69f55c54FB717.json:5]: This pool contains CRV and sdCRV, a tokenized veCRV from StakeDAO.

File: data/meta/tokens/1/0x5ca0313D44551e32e0d7a298EC024321c4BC59B4.json (hunk 2 above)
Comment (DarkGhost7) [reply data/meta/tokens/1/0x5ca0313D44551e32e0d7a298EC024321c4BC59B4.json]: fixed

File: data/meta/tokens/1/0x48fF31bBbD8Ab553Ebe7cBD84e1eA3dBa8f54957.json (hunk 3 above)
Comment (DarkGhost7) [reply data/meta/tokens/1/0x48fF31bBbD8Ab553Ebe7cBD84e1eA3dBa8f54957.json]: fixed

File: data/meta/tokens/1/0x79e281bc69a03dabccd66858c65ef6724e50aebe.json (hunk 4 above)
Comment (DarkGhost7) [reply data/meta/tokens/1/0x79e281bc69a03dabccd66858c65ef6724e50aebe.json]: fixed

File: data/meta/tokens/1/0x8c524635d52bd7b1Bd55E062303177a7d916C046.json (hunk 5 above)
Comment (DarkGhost7) [reply data/meta/tokens/1/0x8c524635d52bd7b1Bd55E062303177a7d916C046.json]: fixed

File: data/meta/tokens/1/0xf7b55C3732aD8b2c2dA7c24f30A69f55c54FB717.json (hunk 6 above)
Comment (DarkGhost7) [reply data/meta/tokens/1/0xf7b55C3732aD8b2c2dA7c24f30A69f55c54FB717.json]: fixed

**Pull request merged**